src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.exception.DukeException;
+
+import duke.taskTypes.Task;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * Command that contains details when deleting a task
+ */
+public class DeleteCommand extends Command {
+    private final String chosenList;
```

Comment:
Is it weird to call this chosenList when you are deleting a task? :flushed:
Maybe you can call it chosenTask or taskNumber?
Same for the doneCommand as well!

Reply:
Yeah misnaming on my part. Thanks for pointing it out!